source/clear-linux/tutorials/gce/gce.rst

```diff
@@ -0,0 +1,122 @@
+.. _clear-on-gce:
+
+Launch |CL-ATTR| Compute Engine on Google Cloud Platform
+########################################################
+
+|CL-ATTR| is a Linux distribution designed for bare mental and virtual machines
+running from the cloud to the edge. This tutorial walks you through the steps
+to create virtual machines in
+`Google Cloud Platform (GCP) <https://cloud.google.com/>`_ running |CL|.
+Reference :ref:`azure` and :ref:`aws-web` for launching |CL| VMs on
+Microsoft\* Azure\* cloud services and Amazon\* Web Services respectively.
+
+#. Google Cloud Platform uses **Projects** to manage resources. Sign in
```

Comment:
I like the reference to the Google Cloud documentation. It would also be nice to have a simple direct link to https://console.cloud.google.com/.

Reply:
Hyperlink added